<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>智慧图书馆</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{border-image: url(:/pic/background.jpg);}
QPushButton {background-color: rgba(204, 204, 204,0.2);padding: 5px 6px 5px 6px;font: 27px 'Microsoft YaHei';letter-spacing: 1px;border: transparent;border-radius: 6px;}
QPushButton:pressed:hover {background-color: rgba(153, 153, 153,0.5);}
QPushButton:hover {background-color: rgba(230, 230, 230,0.4);border: 1px solid gray}
QPushButton:disabled {background-color: rgba(204, 204, 204,0.3);color: rgb(120, 120, 120);}
QLabel{letter-spacing: 1px;}
QPushButton{letter-spacing: 2px;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bluetooth_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>蓝牙设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="ble_attitude">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/pic/red.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="net_attitude">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/pic/red.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sql_attitude">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/pic/red.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>网络连接</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数据库</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>蓝牙连接</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bookMange_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>图书管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>系统设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findBook_Btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>我要找书</string>
    </property>
   </widget>
   <widget class="QPushButton" name="borrowBook_Btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>140</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>我要借书</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnBook_Btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>280</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>我要还书</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewBook_Btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>420</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>图书信息查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reboot_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>重启系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="backLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0.8);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>461</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>By 赵杨磊 叶钰亭 周俊宇</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>backLabel</zorder>
   <zorder>bluetooth_Btn</zorder>
   <zorder>quit_Btn</zorder>
   <zorder>ble_attitude</zorder>
   <zorder>net_attitude</zorder>
   <zorder>sql_attitude</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>bookMange_Btn</zorder>
   <zorder>settings_Btn</zorder>
   <zorder>findBook_Btn</zorder>
   <zorder>borrowBook_Btn</zorder>
   <zorder>returnBook_Btn</zorder>
   <zorder>viewBook_Btn</zorder>
   <zorder>reboot_Btn</zorder>
   <zorder>label_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
